docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -78,14 +78,13 @@ Not applicable as this is the initial release of GDC 2.0.
         * Keyboard focus is not returned to the triggering element when modals are closed.
         * Assistive technologies may not behave correctly with some controls due to incorrect, missing, or redundant labels, attributes, or roles.
 * Cohorts:
-    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](/Data_Portal/Users_Guide/quick_start.md#main-toolbar) locally.
+    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](/Data_Portal/Users_Guide/getting_started.md#main-toolbar) locally.
```

Comment:
preemptively updating links

Reply:
Cool, I've started changing this on my PR, so don't worry about doing this outside of the release notes.